File: Projects/codechef/SEAVOTE.java
package codechef;

import java.util.Arrays;
import java.util.Scanner;

class SEAVOTE {
    private static boolean solve(int[] bs) {
        int nPositives = (int) Arrays.stream(bs).filter(x -> x > 0).count();
        int sum = Arrays.stream(bs).reduce((x1, x2) -> x1 + x2).getAsInt();
        return 100 > sum - nPositives && 100 <= sum;
    }

    public static void main(String[] args) {
        Scanner stdin = new Scanner(System.in);
        int t = stdin.nextInt();
        while (t-- > 0) {
            int n = stdin.nextInt();
            int[] bs = new int[n];
            for (int i = 0; i < n; i++) {
                bs[i] = stdin.nextInt();
            }
            boolean res = solve(bs);
            System.out.println(res ? "YES" : "NO");
        }
        stdin.close();
    }
}

File: Projects/codechef/SEAVOTE.kt
package codechef

import java.util.Arrays

internal object SEAVOTE {
    private fun solve(bs: IntArray): Boolean {
        val nPositives = Arrays.stream(bs).filter { x -> x > 0 }.count() as Int
        val sum: Int = Arrays.stream(bs).reduce { x1, x2 -> x1 + x2 }.getAsInt()
        return 100 > sum - nPositives && 100 <= sum
    }

    fun main(args: Array<String?>?) {
        val stdin = Scanner(System.`in`)
        var t: Int = stdin.nextInt()
        while (t-- > 0) {
            val n: Int = stdin.nextInt()
            val bs = IntArray(n)
            for (i in 0 until n) {
                bs[i] = stdin.nextInt()
            }
            val res = solve(bs)
            System.out.println(if (res) "YES" else "NO")
        }
        stdin.close()
    }
}